src/drawing/DrawingPathGroup.tsx

```diff
@@ -44,9 +44,7 @@ export const DrawingPathGroup = ({
             stroke={color}
             strokeWidth={strokeWidth}
         >
-            {paths.map(({ clientId, points }) => (
-                <DrawingPath key={clientId} borderStrokeWidth={strokeWidthWithBorder} isDecorated points={points} />
-            ))}
+            {children && typeof children === 'function' ? children(strokeWidthWithBorder) : children}
```

Comment:
Checking the existence of `children` is not necessary here, since `typeof undefined === 'undefined'`.